### PR TITLE
DeMarco Gleason

### PR DESCRIPTION
**Pull request by demarcogleason**
# Checkpoint Rubric

This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

## Checkpoint 1: Company/Organization Website following Wireframe
- [x] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 10 points
- [x] A CSS file is effectively linked to the HTML file in the Head section: 10 points
- [x] A link to a CSS reset file is included in the Head section of the HTML file: 10 points 
- [x] The webpage links to another page (multi-page website): 10 points
- [x] The company/orgnanization name is lined up with the navigation menu at the top of the page: 10 points
- [x] The title is centered in front of a background image: 10 points
- [x] The body images are sized and formatted according to the wire frame: 10 points
- [x] The body text is sized, positioned, and formatted to match the fire frame: 10 points
- [x] Flexbox or floats are used to accurately replicate the layout of the wireframe: 10 points
- [x] The footer consists of a background image, centered text, and 2 buttons that function as links: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 2: Foundry Winery Website
- [x] Responsive navbar: 20 points
- [x] Responsive site (resizes to fit desktop, tablet, and mobile views): 30 points 
- [x] Matched layout of original site: 20 points
- [x] Matched font of original site: 10 points
- [x] Properly commented code: 10 points
- [x] Properly indented code: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 3: Learning Team Project
- [ ] Mockup meets customer requirements: 25 points
- [ ] Website meets customer requirements: 25 points
- [ ] Learning team worked together effectively and contributed equally: 25 points
- [ ] Class presentation was engaging and effective: 25 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 4: Personal Portfolio Website
- [ ] Responive: 20 points
- [ ] Evidence of Effective Use of Design Principles: 20 points
- [ ] Links to Resume: 20 points
- [ ] Links to at least 3 Projects: 20 points
- [ ] Effective Communicates Desired Message ("Hire me"): 20 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

### CODE REVIEW

File: 02Lesson/01Homework/index.html [hunk 1]
```diff
@@ -1,8 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title></title>
```
Comment (ghost) [inline 02Lesson/01Homework/index.html:4]: Just testing...

File: 02week/nasa/css/style.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  height:324px;
+  width:850px;
+  background-color:#E6E6E6;
+  margin: 0 auto;
+
+  display: flex;
+  justify-content: space-around;
+}
+
+/*Adjusted the color of each item to match the actual color of each social media icon*/
```
Comment (RFloydUT3) [inline 02week/nasa/css/style.css:52]: Great idea adding the social media icons/ proper colors

File: 02week/nasa/index.html [hunk 3]
```diff
@@ -1,11 +1,75 @@
...
+            <h1>NASA</h1>
+        </section>
+        <br>
+
+
+
+        <!--Contact Info-->
+        <section id="contact-info">
+            <div class="social-media">
+                <h2><strong>Social Media</strong></h2>
```
Comment (lau-marquez) [inline 02week/nasa/index.html:29]: cool addition of icons!

File: 02week/nasa/css/style.css [hunk 4]
```diff
@@ -0,0 +1,97 @@
+/*Helps me visualize the how the padding and margin of elements will affect various surrounding elements*/
+/**{
+  border: solid red .2em;
+}*/
+
+/* modifies the base font-size on the body using a percentage. This adjusts things so that 1em equals 10px, instead of the default 16px. To set the font-size to the equivalent of 14px, set it to 1.4em.
+*/
+body { font-size:62.5%; }
+h1 { font-size: 2.4em; } /* =24px */
```
Comment (RFloydUT3) [inline 02week/nasa/css/style.css:9]: didnt realize you do not have to use "px" to adjust size

File: 03week/planet/index.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+          <span><a href="#Economy">Economy</a></span>
+        </nav>
+</header>
+
+    <main><h3><strong>Giannis Antetokounmpo: The Most Intriguing Point Guard In NBA History</strong></h3>
+        <section>
+
+            <article>
+              <img src="img/bucks.jpg" alt="Giannis layup" class="bucks"><p>Antetokounmpo stands 6' 11", with legs so long opposing coaches constantly complain that he is traveling, until they review the tape. “He’s not,” says Wizards coach Scott Brooks. “It’s just that we’ve never seen somebody with a stride like this.” Among the NBA’s legion of stretchy giants, Kevin Durant is the scorer, Anthony Davis the slasher. Antetokounmpo is the creator, traversing half the court with four Sasquatch steps, surveying traffic like a big rig over smart cars. Durant and Davis try to play point guard. Antetokounmpo actually does it, dropping dimes over and around defenders’ heads, leading the Bucks in every major category; 23.8 points, 8.9 rebounds, 5.9 assists, 2.0 blocks and 2.0 steals. This season he will be the team's first All-Star since Michael Redd in 2004, and before you learn to spell his surname, he will be much more.</p>
```
Comment (Enogelmeier) [inline 03week/planet/index.html]: Hey DeMarco, your page looks good and it seems like all the CSS elements work. I can't resize my img in my body. What am I doing wrong?

**Pull request closed without merging**